docker/docker-entrypoint.sh

```diff
@@ -39,6 +39,10 @@ if not User.objects.filter(username='${SUPERUSER_NAME}'):
     Token.objects.create(user=u, key='${SUPERUSER_API_TOKEN}')
 END
 
+for script in startup_scripts/*.py; do
+  ./manage.py shell --plain < $script
```

Comment:
The files are launched every time the container starts, right?

Reply:
You should put quotation marks around $script, in case a file is saved with a space in the name:

`"${script}"`

Reply:
👍